src/test/resources/templates/V102__Trigger_template_duplicate_tag.sql

```diff
@@ -0,0 +1,73 @@
+use cfe_18;
+
+/*
+ This is a template for cant_add_existing_tag_into_group in before_insert_triggers.sql file
+
+ Workflow
+    1. get all tags
+    2. link tags according to a capture_group
+    3. compare capture_groups against host_groups
+    4. check all hosts in that group to see if they exist already on said tag
+ */
+
+-- valitaan kaikki tagit pöytään
```

Comment:
use english and declare these as examples of the idea behind g_x_g triggers

Reply:
Fixed